fishy/helper/hotkey/hotkey_process.py

```diff
@@ -63,7 +64,7 @@ def _event_loop(self):
             if key in Key:
                 callback = self._hotkeys[key]
                 if callback:
-                    playsound(helper.manifest_file("beep.wav"), False)
+                    playsound(helper.manifest_file("https://raw.githubusercontent.com/fishyboteso/fishyboteso/main/fishy/beep.wav"), False)  # Added comma here
```

Comment:
y fetch it from web, instead of using the locally stored file?

Reply:
No, this is a test, I am working on it